spec/system/polis_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "pol.is", type: :system do
+  include_context "with a component"
```

Comment:
Please use a fixed page_id as defined here: `Decidim::Polis::PolisController#page_id`
```
"#{current_component.id}#{params[:participatory_process_slug]}"
```
in order to not generate polis discussions